pyproject.toml

```diff
@@ -52,18 +51,9 @@ optional-dependencies = {docs = [
     "napari-plugin-engine>=0.1.4",
     "napari>=0.4.16",
     "qtpy",
-], pyqt = [
-    "PyQt5 >= 5.12.3, != 5.15.0",
-    "btrack[napari]",
-], pyside = [
-    "PySide2 >= 5.13.2, != 5.15.0 ; python_version != '3.8'",
-    "PySide2 >= 5.14.2, != 5.15.0 ; python_version == '3.8'",
-    "btrack[napari]",
-], qt = [
-    "btrack[pyside]",
```

Comment:
I think it's better to just let users install their own backend

Reply:
It looks like at some point we wanted to avoid Qt version 5.15.0. Do we remember why?
And are we happy risking it now?

Reply:
These were copied from the `napari` repo https://github.com/napari/napari/blob/48626e8dc369b8c349a30e46167a91cead3ea8c8/setup.cfg#L92-L110

In reality, we'll want users to do `pip install napari[all] btrack[napari]` so that would already be handled